Gemfile

```diff
@@ -22,6 +22,6 @@ group :test do
 end
 
 gem 'prometheus-client', '~> 1.0'
-gem 'prometheus_gcstat', git: 'git@github.com:gocardless/prometheus_gcstat_ruby'
+# gem 'prometheus_gcstat', git: 'git@github.com:gocardless/prometheus_gcstat_ruby'
```

Comment:
This is a private gem. We would need to make this repo private, to run the specs. Bundle install will fail otherwise

Reply:
Can we refer to the gem, rather than the repo? I believe you don't need a public repo for that.